suite/tests/linux/syscall_pwait.cpp

```diff
@@ -197,7 +197,8 @@ main(int argc, char *argv[])
     auto psyscall_epoll_pwait = [test_set](bool nullsigmask) -> int {
         int epoll_fd = epoll_create1(EPOLL_CLOEXEC);
         struct epoll_event events;
-        return epoll_pwait(epoll_fd, &events, 24, -1, nullsigmask ? NULL : &test_set);
+        return epoll_pwait(epoll_fd, &events, IF_X64_ELSE(24LL, 24L),
```

Comment:
The man page says this is "int maxevents" so this looks like a bad change, with the original correct.

Reply:
You're right, plus the non variadic argument functions know their types anyway. Reverted this and the next. I went through all the syscall() instances in the test again and now cast into their exact types.